distribution/docs/src/main/resources/content/_configuring/banner-markings-content-extractor.adoc

```diff
@@ -0,0 +1,88 @@
+:title: Banner Markings Extractor
+:type: configuringConfigFile
+:status: published
+:summary: Text file banner markings.
+:order: 132
+
+The Banner Markings Extractor parses the text of PDFs, Microsoft Office documents, plain text files,
+ or any other file from which the <<_tika_input_transformer,TikaTransformer>> can extract text. The
+ ${branding} default behavior is to apply high-water security settings when ingesting a document.
+ This is sufficient for documents with no security markings, but is incorrect when working with
+ classified documents that include security mark-up in the text.
+
+The marking extractors parses a banner marking and attempts to create new attributes on the
+metacard of the ingested document. The extractor looks at the first non-blank line of text
+in the document and tries to interpret it as a banner marking. The interpretation of a banner
+marking is guided by DoD Instruction 5200.01 and the ${branding} catalog taxonomy. The marking extractors
+will attempt to get as much information from the banner marking as possible.
+The attributes they attempt to extract are:
+
+* Classification
+* Releasibility
+* Codewords
+* Dissemination Controls
+* Owner-Producer
+* Classification System
+* Special Access Program
+* Atomic Energy Act
+* DoD Unclassified Controlled Nuclear Information (DoD UCNI)
+* Department of Energy (DOE) UCNI
+* Foreign Government Information
+* Other dissemination markings
+
+
+===== Related Components
+
+<<metacard_attribute_security_policy_plugin, Metacard Attribute Security Policy Plugin>>
```

Comment:
this _might_ need a leading underscore for the link. I haven't checked, but that's the convention for those links.